docs/en/modules/adc.md

```diff
@@ -5,9 +5,37 @@
 
 The ADC module provides access to the in-built ADC.
 
-On the ESP8266 there is only a single-channel, which is multiplexed with the battery voltage. Depending on the setting in the "esp init data" (byte 107) one can either use the ADC to read an external voltage, or to read the system voltage, but not both.
+On the ESP8266 there is only a single-channel, which is multiplexed with the battery voltage. Depending on the setting in the "esp init data" (byte 107) one can either use the ADC to read an external voltage, or to read the system voltage (vdd33), but not both.
 
-The default setting in the NodeMCU firmware can be controlled via user_config.h at compile time, by defining one of ESP_INIT_DATA_ENABLE_READVDD33, ESP_INIT_DATA_ENABLE_READADC or ESP_INIT_DATA_FIXED_VDD33_VALUE. To change the setting at a later date, use Espressif's flash download tool to create a new init data block.
+Which mode to use the ADC in can be configured via the `adc.force_init_mode()` function. Note that after switching from one to the other a system restart is required before the change takes effect.
+
+## adc.force_init_mode()
+
+Checks and if necessary reconfigures the ADC mode setting in the ESP init data block.
+
+####Syntax
+`adc.force_init_mode(mode_value)`
+
+####Parameters
+`mode_value` One of `adc.INIT_ADC` or `adc.INIT_VDD33`.
+
+####Returns
+True if the function had to change the mode, false if the mode was already configured. On a true return the ESP needs to be restarted for the change to take effect.
```

Comment:
True if the function had to change the mode, false if the mode was already configured. On a true return the ESP needs to be restarted by power cycling for the change to take effect.